.github/workflows/full-ci.yml

```diff
@@ -101,7 +101,7 @@ jobs:
       matrix_compiler_cc: 'gcc-10'
       matrix_compiler_cxx: 'g++-10'
       timeout: 120
-      bootstrap_args: '--enable-serialization'
+      bootstrap_args: '--enable-serialization --vcpkg-target-triplet=x64-linux-asan'
```

Comment:
This test subverts any test of the actual bootstrap interface by not using `--enable-sanitizer` and not specifying the base triplet.

Reply:
Switched to specify the base triplet. `--enable-sanitizer` is passed in https://github.com/teo-tsirpanis/TileDB/blob/425fd620df535b5d319bbb0ca917a0d27c6830c0/.github/workflows/ci-linux_mac.yml#L66.